### DISCUSSION TITLE
Docker SSL - any way to force enable it?

### DISCUSSION THREAD

**Discussion by git-Contributor** [jonaswinkler/paperless-ng#1363] [Support]
Hi,
I know SSL was (sadly) removed from docker in [0.9.0](https://paperless-ng.readthedocs.io/en/latest/changelog.html?highlight=ssl#paperless-ng-0-9-0), but is there still any option to enable it somehow - e.g. via some config files inside the docker or any other way? 

It is really not good practise to have all sensitive documents stored via unsecured app. And proxy server really does not solved this, it basically just hides the issue, but the communication is still unsecured (between proxy and paperless).

**Comment by TobenderZephyr** [comment]
Actually you are partly right - as long as proxy and paperless-ng were running on different hosts.
But if you are running both on the same machine like most of us (have a look at traefik for example), this does not matter:
You would be able to sniff network packets only if you are logged in on the machine (between both network interfaces). But once you are there, it is too late anyway.

Don't run paperless-ng on the internet (portforwarding) and you are (quite a good amount) safe.

But this applies to consume folders which you share on your network as well.
